### ISSUE TITLE
1920x1200

### ISSUE THREAD

**Issue by jatolimon** [Jay-Tech/ioSenderTouch#12]
Hi! 
is there a manual way of changing the resolution of the UI. If I change windows resolution to 1920x1080 and back to 1920x1200 while IOT is open it fixes it. But only until next time I open it. 
Release is 1.0.1.5
![image](https://github.com/user-attachments/assets/876ecce5-c0af-47f2-9367-d3257a12df83)
Regards,
Pablo.

**Comment by Jay-Tech** [comment]
yeah, two ways one is to enable tool bar then maximize  window (app settings -> Window tool bar -> save settings -> restart app) or
![image](https://github.com/user-attachments/assets/4e41e7d1-5516-401b-bbb4-05cc6a013ecd)

open the app setting file and at the very bottom change the resloution if you want to run without the window tool bar (like in you screenshot)
![image](https://github.com/user-attachments/assets/fd7e6602-835a-4c55-a2e2-765a67887677)

**Comment by jatolimon** [comment]
Thank you! That worked
